src/runners/http_rfqm_service_runner.ts

```diff
@@ -146,6 +149,9 @@ export async function buildRfqmServiceAsync(connection: Connection, asWorker: bo
 
     const quoteServerClient = new QuoteServerClient(axiosInstance);
 
+    const redisClient = redis.createClient({ url: REDIS_URI });
```

Comment:
Will this throw an error when `REDIS_URI === undefined`?

Reply:
Currently playing around w/this variable, but will make sure to make it fail early if necessary!

Reply:
Alright! updated this with a sane default (localhost, which is the same default as we use for postgres)